### PR TITLE
View more Agg toggle under row chart

### PR DESCRIPTION
**Pull request by flacoman91**
adding this thing under the row chart

Screen Shot 2020-06-22 at 7 41 26 AM

- removed total from external tooltip when line chart selected

view-more

### CODE REVIEW

File: src/components/Trends/TrendDepthToggle.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+      }
+      return <div className={ 'trend-depth-toggle' }>
+        <button className={ 'a-btn a-btn__link' }
+                id={ 'trend-depth-button' }
+                onClick={ () => {
+                  this.props.resetDepth()
+                } }>
+          <span className={ 'minus' }></span>
+          { `View ${ resetCount } fewer ${ lens.toLowerCase() }s` }
```
Comment (JeffreyMFarley) [inline src/components/Trends/TrendDepthToggle.jsx:30]: "View 13 fewer" feels awkward to me.  Was this in Matt's comp?  I would think "View only 5" or simply "View fewer" would be better

File: src/reducers/query.jsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export const defaultQuery = {
   subLens: '',
   tab: types.MODE_MAP,
   totalPages: 0,
+  trend_depth: 5,
```
Comment (JeffreyMFarley) [inline src/reducers/query.jsx:34]: 🐫  case or 🐍  case?

Comment (flacoman91) [reply src/reducers/query.jsx]: should be camel case for JS standards.  the api wants the 🐍  though

File: src/components/Trends/TrendDepthToggle.jsx (hunk 1 above)
Comment (flacoman91) [reply src/components/Trends/TrendDepthToggle.jsx:30]: We decided on Show More / Show Fewer in the call.

**Pull request merged**